addon/components/hyper-table-v2/filtering-renderers/common/search.hbs

```diff
@@ -1,18 +1,10 @@
 <div data-control-name={{concat "hypertable__column_filtering_for_" @column.definition.key "_filters"}}>
   <div class="margin-top-xxx-sm">
     <label>{{t "hypertable.column.filtering.search_term.label"}}</label>
-    <OSS::InputContainer
+    <OSS::SearchField
       @value={{this.searchQuery}}
       @onChange={{this.onInputChanged}}
       @placeholder={{t "hypertable.column.filtering.search_term.placeholder"}}
-      {{(if (eq @type "number") (modifier "did-insert" this.setupOnlyNumericListener))}}
-      {{(if (eq @type "number") (modifier "will-destroy" this.teardownOnlyNumericListener))}}
-    >
-      <:suffix>
-        {{#if (gt this.searchQuery.length 0)}}
-          <OSS::Icon @icon="fa-times" role="button" {{on "click" this.onClearSearch}} />
-        {{/if}}
-      </:suffix>
-    </OSS::InputContainer>
+    />
```

Comment:
You can add a data-control-name here :) QA will be happy 😇 
`data-control-name="filter-renderers-common-search-input"` (used dashes instead of underscores because I think we have more usages with dashes throughout the platform. Up to you really!)